scripts/deploy-database.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+cleanup() {
+	# kill proxy
+	kill -INT "$proxy_program_PID"
+	sleep 0.4
+}
+
+# trap exit of script and call cleanup()
+trap cleanup EXIT
+
+# create proxy to connect to database on port 5432
+coproc proxy_program (flyctl proxy 5432 -a mboyea-database-test)
+
+# load .env
+set -a; source .env; set +a
+
+# wait for output from proxy program
+read output <&"${proxy_program[0]}"
+# print output from proxy program
```

Comment:
There's more of the same inconsistent spacing when it comes to comments. I prefer to add the extra spaces all the time; consider doing the same

Reply:
Good points about inconsistent spacing